pom.xml

```diff
@@ -275,6 +273,17 @@
     </repository>
   </repositories>
   <pluginRepositories>
+    <pluginRepository>
```

Comment:
Yep. +1 for piggybacking this. I also agree that this is required to unblock https://github.com/apache/spark/pull/27279 .